TRB_label_assist/SVM_decision_boundaries.ipynb

```diff
@@ -284,7 +283,7 @@
             "name": "python",
             "nbconvert_exporter": "python",
             "pygments_lexer": "ipython3",
-            "version": "3.9.16"
```

Comment:
Ditto

Reply:
different python version than the one used by person who committed this previously. This shows up as 3.9.6 on the json of the notebook as soon as I run even 1 cell. I usually reset this before commits to not create unnecessary history. This might have slipped passed me in one of the previous commits.